controllers/users.controller.js

```diff
@@ -0,0 +1,54 @@
+const { OAuth2Client } = require("google-auth-library");
+const jwt = require("jsonwebtoken");
+const User = require("../models/User");
+
+exports.login = async (req, res, next) => {
+  try {
+    const googleToken = req.headers.authorization.split(" ")[1];
+    const client = new OAuth2Client(process.env.GOOGLE_CLIENT_ID);
+    const ticket = await client.verifyIdToken({
+      idToken: googleToken,
+      audience: process.env.GOOGLE_CLIENT_ID,
+    });
+
+    const { name, email, picture } = ticket.getPayload();
+
+    let user = await User.findOne({ email });
+
+    if (!user) {
+      user = await User.create({
+        name,
+        email,
+        money: 5000,
+        image_url: picture,
+      });
+    }
+
+    const token = jwt.sign({
+      _id: user._id,
+      name: user.name,
+      email: user.email,
+    }, process.env.SECRET_KEY, {
+      expiresIn: "7d",
+    });
```

Comment:
```suggestion
    const token = jwt.sign(
      {
        _id: user._id,
        name: user.name,
        email: user.email,
      }, 
      process.env.SECRET_KEY,
      {
        expiresIn: "7d",
      }
    );
```
음.. 이건 뭐가 맞는 건지 모르겠어요 Frontend에서는 전 이런 식으로 쓴 부분이 있어서...
이따 얘기해봐욥